lib/vfscore/automount.c

```diff
@@ -344,6 +392,20 @@ UK_LIBPARAM_PARAM_ARR_ALIAS(fstab, &fstab_up, charp,
 
 #endif /* CONFIG_LIBVFSCORE_AUTOMOUNT_UP */
 
+int vfscore_get_initrd0(struct ukplat_memregion_desc **out)
```

Comment:
This seems to only be called in this file. Maybe we should make it `static?

Reply:
true, can be `static`, will adopt